tests/Controller/ChannelApiTest.php

```diff
@@ -102,6 +192,26 @@ public function it_allows_showing_channel()
         $this->assertResponse($response, 'channel/show_response', Response::HTTP_OK);
     }
 
+    /**
+     * @test
+     */
+    public function it_allows_to_delete_channel()
```

Comment:
It would be also great to have test like `it_does_not_allow_to_delete_non_existing_channel`.